data/alpha/register/social-housing-provider-legal-entity.yaml

```diff
@@ -0,0 +1,9 @@
+fields:
+- social-housing-provider-legal-entity
```

Comment:
but social-housing-providerS-legal-entity

i.e. legal entity that belongs to social housing provider.

This is another thing where you're gonna say "but thats the way the api works" and i'm gonna say "I think the api is wrong" but we're gonna merge it anyway because thats how we do. 😂